src/test/java/io/strimzi/kafka/bridge/http/ProducerTest.java

```diff
@@ -191,7 +191,7 @@ void sendSimpleMessageWithKey(VertxTestContext context) throws InterruptedExcept
     }
 
     @Disabled("Will be check in the next PR, this is just external tests for Bridge")
-    @DisabledIfEnvironmentVariable(named = "BRIDGE_EXTERNAL_ENV", matches = "((?i)FALSE(?-i))")
+    @DisabledIfEnvironmentVariable(named = "EXTERNAL_BRIDGE", matches = "((?i)TRUE(?-i))")
```

Comment:
wasn't disabling test supposed to be in a different PR?

Reply:
I have changed the logic and naming the env nothing more.